### DISCUSSION TITLE
In the Frigate+ interface, specifically for the local section used to upload to the cloud, can we use CLI through PuTTY?

### DISCUSSION THREAD

**Discussion by mitrejit** [blakeblackshear/frigate#13626] [General]
In the Frigate+ interface, specifically in the local section for uploading to the cloud, can we use the CLI through PuTTY? I have uploaded 4144 images to request the model already, and now Frigate detects fewer false positives, but there are still some. I would like to upload images all at once using the CLI because using the GUI and clicking each image with the mouse is quite slow. Is this possible?  
![001](https://github.com/user-attachments/assets/da293242-03a9-4675-8777-562605342085)
![002](https://github.com/user-attachments/assets/2067253e-0369-4cb9-842c-2185934cfc57)

**Comment by hawkeye217** [comment]
No, this is not possible.

**Reply by NickM-27** [reply]
Some users have done this, but you'd need to look at the code to get an auth token and run the API by hand
